AnkiDroid/src/main/java/com/ichi2/anki/servicelayer/scopedstorage/migrateuserdata/MigrateUserData.kt

```diff
@@ -588,6 +603,37 @@ open class MigrateUserData protected constructor(val source: Directory, val dest
 
         return true
     }
+
+    /**
+     * Migrate a file to [expectedFileLocation] if it exists inside [source]
+     * @param expectedFileLocation A file which exists inside [destination]
```

Comment:
```suggestion
     * @param expectedFileLocation A file which should exist inside [destination]
```

Since the file is expected to exist there but doesn't exist as of now.